packages/generator/templates/ClientInstance.ts.hbs

```diff
@@ -1,15 +1,9 @@
-
+/* eslint-disable */
 import { StaticClient, BaseClient, ClientConfig, Response, ResponseError } from "@commerce-apps/core";
 import _ from "lodash";
 import { OperationOptions } from "retry";
 
-import {
-{{#each dataTypes}}
-    {{#if (isTypeDefinition .)}}
-        {{getValue name}}T,
-    {{/if}}
-{{/each}}
-} from "./{{apiSpec}}.types";
+import * as types from "./{{apiSpec}}.types";
```

Comment:
Does removing this conditional change the logic in a meaningful way?

Reply:
I couldn't see another way to import the content of a folder "as types". I can have another look if you don't like this..